### PR TITLE
update doc for defacing using R

### PR DESCRIPTION
**Pull request by alexfoias**
Not sure if the formatting of the code block is correct. I tried the preview in VS code and it looks okay.

### CODE REVIEW

File: processing/deface_spineGeneric_usingR.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        for subject in (os.listdir(output_path)):
+            pathSubject = os.path.join(output_path,subject)
+            if os.path.isdir(pathSubject):
+                for contrast in contrastList:
+                    pathContrastDefaced = os.path.join(pathSubject,'anat',subject+'_'+contrast+ '_defaced.nii.gz')
+                    pathContrast = os.path.join(pathSubject,'anat',subject+'_'+contrast+ '.nii.gz')
+                    if not os.path.isfile(pathContrastDefaced) and ((subject+'_'+contrast+ '_defaced.nii.gz') not in exclude_list):
+                        print ('Currently processing: ' + pathContrast )
+                        try:
+                            command = 'Rscript '+ currentwd + '/regular_deface.r -i '+pathContrast+ ' -o '+pathContrastDefaced
```
Comment (jcohenadad) [inline processing/deface_spineGeneric_usingR.py:43]: implies that this script needs to be run where regular_deface is located-- make it explicit

File: docs/documentation.rst [hunk 2]
```diff
@@ -166,7 +166,24 @@ defaced subject:
 
    Example of manual defacing.
 
+Defacing using R
```
Comment (jcohenadad) [inline docs/documentation.rst:169]: should flow naturally with previous section where you talk about defacing-- the flow is illogical as implemented here-- previous section says: mri_deface doesn't work, so we recommend to do it manually-- and then this paragraph appears-- please work on the transition/overall logic of the sectinons

Comment (alexfoias) [reply docs/documentation.rst:158]: fixed in https://github.com/sct-pipeline/spine-generic/pull/56/commits/aa153c59aef0156f98d5bc0c355d05ae66f2bb4a

File: processing/deface_spineGeneric_usingR.py (hunk 1 above)
Comment (alexfoias) [reply processing/deface_spineGeneric_usingR.py:49]: fixed in https://github.com/sct-pipeline/spine-generic/pull/56/commits/7535e677fd7fed359876e11cc300ac25f6a49606 and https://github.com/sct-pipeline/spine-generic/pull/56/commits/54cd4f6c0cda0c0487bdc08493326195167aa1a4

**Pull request merged**